tests/core/test_daemon_rpc.py

```diff
@@ -1,20 +1,21 @@
+# flake8: noqa: F821
+# pylint: disable=E0602
```

Comment:
Python fixture auto-imports are not recognized by flake / pylint / black

Reply:
you're removing these, right?